provider/identity_provider_resource.go

```diff
@@ -507,6 +643,7 @@ func (r *IdentityProviderResource) Read(ctx context.Context, request tfsdk.ReadR
 	gitlabObject := object.Gitlab()
 	ldapObject := object.LDAP()
 	openidObject := object.OpenID()
+	githubObject := object.Github()
```

Comment:
Github -> GitHub

Reply:
This is the exported method name coming from github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1